Dockerfile

```
@@ -1,11 +1,9 @@
 # This Dockerfile starts the entrypoint script to evaluate if a new moodle version exists and an update should be started.
 
 # Stage 1: Build stage
-FROM bitnami/moodle:4.1.12-debian-12-r1 AS build
+FROM bitnami/moodle:4.1.13-debian-12-r2 AS build
```

since the moodle version increased it would be 
```suggestion
FROM bitnami/moodle:4.1.13-debian-12-r1 AS build
```